RELEASE/scripts/autoscend.ash

```diff
@@ -147,6 +148,7 @@ void initializeSettings() {
 	set_property("auto_beatenUpLastAdv", false);
 	remove_property("auto_beatenUpLocations");
 	set_property("auto_getBeehive", false);
+	set_property("auto_bowledAtAlley", false);
```

Comment:
is this once per ascension? it looks like it is. in which case it should probably not be a boolean but an int of my_ascensions()

Reply:
Also should probably have a different name to indicate it is referring to the iotm bowling ball and not the regular ones